Fix mistakes in French translation

paulus85 opened the pull request:
* This PR fixes several mistakes in the French translation on lazydocker.

ghislaindurand commented on pkg/i18n/french.go:
```diff
@@ -76,30 +76,30 @@ func frenchSet() TranslationSet {
 		VolumesTitle:              "Volumes",
-		CustomCommandTitle:        "Commande personalisé:",
-		BulkCommandTitle:          "Commande groupée:",
+		CustomCommandTitle:        "Commande personalisée :",
```
CustomCommandTitle: "Commande personnalisée :",

pablomalo commented on pkg/i18n/french.go:
```diff
@@ -8,32 +8,32 @@ func frenchSet() TranslationSet {
 		ConnectionFailed:           "Erreur lors de la connexion au client docker. Essayez de redémarrer votre client docker",
-		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez executer le service avec le drapeau 'it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yaml",
-		WaitingForContainerInfo:    "Le processus ne peut pas continuer avant que docker donne plus d'informations. Veuillez réessayer dans quelques instants.",
+		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez exécuter le service avec le drapeau 'it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yaml",
```
```suggestion
		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez exécuter le service avec le drapeau '-it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yml",
```
Par souci d'homogénéité avec la version anglaise. 😄

pablomalo commented on pkg/i18n/french.go:
```diff
@@ -8,32 +8,32 @@ func frenchSet() TranslationSet {
-		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez executer le service avec le drapeau 'it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yaml",
-		WaitingForContainerInfo:    "Le processus ne peut pas continuer avant que docker donne plus d'informations. Veuillez réessayer dans quelques instants.",
+		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez exécuter le service avec le drapeau 'it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yaml",
+		WaitingForContainerInfo:    "Le processus ne peut pas continuer avant que docker ne donne plus d'informations. Veuillez réessayer dans quelques instants.",
```
```suggestion
		WaitingForContainerInfo:    "Le processus ne peut pas continuer avant que docker ne fournisse plus d'informations. Veuillez réessayer dans quelques instants.",
```
Un peu plus soutenu ?

pablomalo commented on pkg/i18n/french.go:
```diff
@@ -8,32 +8,32 @@ func frenchSet() TranslationSet {
 		CannotAttachStoppedContainerError: "Vous ne pouvez pas vous attacher à un conteneur en arrêt, vous devez le démarrer avant (ce que vous pouvez faire avec la touche 'r') (oui, je suis trop paresseux pour le faire automatiquement pour vous) (plutôt cool que je puisse communiquer en tête-à-tête avec vous au travers d'un message d'erreur)",
-		CannotAccessDockerSocketError:     "Impossible d'accèder à la socket docker à: unix:///var/run/docker.sock\nLancez lazydocker en tant que root ou alors lisez https://docs.docker.com/install/linux/linux-postinstall/",
-		CannotKillChildError:              "Trois secondes ont étés attendu pour l'arrêt des processus fils. Il y a possiblement un processus orphelin qui continue à tourner sur votre systeme.",
+		CannotAccessDockerSocketError:     "Impossible d'accéder à la socket docker à: unix:///var/run/docker.sock\nLancez lazydocker en tant que root ou alors lisez https://docs.docker.com/install/linux/linux-postinstall/",
```
```suggestion
		CannotAccessDockerSocketError:     "Impossible d'accéder au socket docker à : unix:///var/run/docker.sock\nLancez lazydocker en tant que root ou lisez https://docs.docker.com/install/linux/linux-postinstall/",
```
Il me semble que parle généralement d'une soquette 🧦 mais d'_un_ socket. 😉

pablomalo commented on pkg/i18n/french.go:
```diff
@@ -8,32 +8,32 @@ func frenchSet() TranslationSet {
-		CannotAccessDockerSocketError:     "Impossible d'accèder à la socket docker à: unix:///var/run/docker.sock\nLancez lazydocker en tant que root ou alors lisez https://docs.docker.com/install/linux/linux-postinstall/",
-		CannotKillChildError:              "Trois secondes ont étés attendu pour l'arrêt des processus fils. Il y a possiblement un processus orphelin qui continue à tourner sur votre systeme.",
+		CannotAccessDockerSocketError:     "Impossible d'accéder à la socket docker à: unix:///var/run/docker.sock\nLancez lazydocker en tant que root ou alors lisez https://docs.docker.com/install/linux/linux-postinstall/",
+		CannotKillChildError:              "Trois secondes ont étés attendues pour l'arrêt des processus fils. Il y a possiblement un processus orphelin qui continue à tourner sur votre système.",
```
```suggestion
		CannotKillChildError:              "Trois secondes se sont écoulées depuis la demande d'arrêt des processus enfants. Il se peut qu'un processus orphelin continue à tourner sur votre système.",
```
"Processus enfants" : à la fois plus lisible et moins "genré" que "processus fils" ?

pablomalo commented on pkg/i18n/french.go:
```diff
@@ -76,30 +76,30 @@ func frenchSet() TranslationSet {
 		StandaloneContainersTitle: "Conteneurs Autonomes",
```
```suggestion
		StandaloneContainersTitle: "Conteneurs autonomes",
```
Pour tous les titres, je serais partisan de ne mettre de majuscule qu'au premier mot, comme il est généralement d'usage en français.

![image](https://github.com/jesseduffield/lazydocker/assets/25877142/8f80c831-e558-4693-a134-0578876c5fba)

pablomalo commented on pkg/i18n/french.go:
```diff
@@ -8,32 +8,32 @@ func frenchSet() TranslationSet {
+		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez exécuter le service avec le drapeau 'it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yaml",
+		WaitingForContainerInfo:    "Le processus ne peut pas continuer avant que docker ne donne plus d'informations. Veuillez réessayer dans quelques instants.",
 
 		CannotAttachStoppedContainerError: "Vous ne pouvez pas vous attacher à un conteneur en arrêt, vous devez le démarrer avant (ce que vous pouvez faire avec la touche 'r') (oui, je suis trop paresseux pour le faire automatiquement pour vous) (plutôt cool que je puisse communiquer en tête-à-tête avec vous au travers d'un message d'erreur)",
```
```suggestion
		CannotAttachStoppedContainerError: "Vous ne pouvez pas vous attacher à un conteneur arrêté, vous devez auparavant le démarrer (ce que vous pouvez faire avec la touche 'r') (oui, je suis trop paresseux pour le faire automatiquement pour vous) (plutôt cool que je puisse communiquer en tête-à-tête avec vous au travers d'un message d'erreur, cependant)",
```

pablomalo commented on pkg/i18n/french.go:
```diff
@@ -76,30 +76,30 @@ func frenchSet() TranslationSet {
+		CustomCommandTitle:        "Commande personalisée :",
+		BulkCommandTitle:          "Commande groupée :",
 		ErrorTitle:                "Erreur",
 		LogsTitle:                 "Enregistrements",
```
```suggestion
		LogsTitle:                 "Journaux",
```
On traduit généralement "system logs" par "journaux système", par exemple.

pablomalo reviewed (changes requested):
Thank you for these much-needed corrections. I hope your PR get approved soon. I took the liberty of making a few additional suggestions. Do not hesitate to discuss them with me if you disagree. 😄

paulus85 commented:
@pablomalo Thanks for your feedbacks! I corrected my PR with your suggestions (and rebased the branch, as new translations have been added in the meantime).

Feel free to review again!

pablomalo commented on pkg/i18n/french.go:
```diff
@@ -54,64 +54,64 @@ func frenchSet() TranslationSet {
 		StopService:                "Êtes-vous certain de vouloir arrêter le conteneur de ce service ?",
 		StopContainer:              "Êtes-vous certain de vouloir arrêter ce conteneur ?",
-		PressEnterToReturn:         "Appuiez sur Enter pour revenir à lazydocker (ce message peut être désactivé dans vos configurations en appliquant `gui.returnImmediately: true`)",
+		PressEnterToReturn:         "Appuyez sur Enter pour revenir à lazydocker (ce message peut être désactivé dans vos configurations en appliquant `gui.returnImmediately: true`)",
```
Appuyez sur Entrée ? 😄

jesseduffield commented:
Great work @paulus85 !
